src/Workspaces/Remote/ServiceHub/Shared/RoslynJsonConverter.cs

```diff
@@ -20,52 +20,54 @@ private AggregateJsonConverter()
         }
 
         public override bool CanConvert(Type objectType)
-        {
-            return _map.ContainsKey(objectType);
-        }
+            => _map.ContainsKey(objectType);
 
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
-        {
-            return _map[objectType].ReadJson(reader, objectType, existingValue, serializer);
-        }
+            => _map[objectType].ReadJson(reader, objectType, existingValue, serializer);
 
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
-        {
-            _map[value.GetType()].WriteJson(writer, value, serializer);
-        }
+            => _map[value.GetType()].WriteJson(writer, value, serializer);
 
-        // this type is shared by multiple teams such as Razor, LUT and etc which have either separated/shared/shim repo
-        // so some types might not available to those context. this partial method let us add Roslyn specific types
-        // without breaking them
+        // this type is shared by multiple teams such as Razor, LUT and etc which have either 
+        // separated/shared/shim repo so some types might not available to those context. this 
+        // partial method let us add Roslyn specific types without breaking them
         partial void AppendRoslynSpecificJsonConverters(ImmutableDictionary<Type, JsonConverter>.Builder builder);
 
         private ImmutableDictionary<Type, JsonConverter> CreateConverterMap()
         {
             var builder = ImmutableDictionary.CreateBuilder<Type, JsonConverter>();
 
-            builder.Add(typeof(Checksum), new ChecksumJsonConverter());
-            builder.Add(typeof(SolutionId), new SolutionIdJsonConverter());
-            builder.Add(typeof(ProjectId), new ProjectIdJsonConverter());
-            builder.Add(typeof(DocumentId), new DocumentIdJsonConverter());
-            builder.Add(typeof(TextSpan), new TextSpanJsonConverter());
-            builder.Add(typeof(SymbolKey), new SymbolKeyJsonConverter());
+            Add(builder, new ChecksumJsonConverter());
+            Add(builder, new SolutionIdJsonConverter());
+            Add(builder, new ProjectIdJsonConverter());
+            Add(builder, new DocumentIdJsonConverter());
+            Add(builder, new TextSpanJsonConverter());
+            Add(builder, new TextChangeJsonConverter());
```

Comment:
❔ This one (`TextChangeJsonConverter`) existed previously but just wasn't registered? Is it possible we no longer need it?